device/mellanox/x86_64-mlnx_msn4410-r0/platform_wait

```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/platform_wait
```

Comment:
are you sure? i prefer in any case it will be using the sn4700 even for visuality

Reply:
This script is common for all our platforms, currently only have the file in the 2700 folder and other platforms have symbol links. The idea is to reduce maintenance efforts.